js/main.js

```diff
@@ -2,13 +2,21 @@
 /* exported data */
 var $photoURL = document.querySelector('.photoURL');
 var $image = document.querySelector('.image');
+var $entryForm = document.querySelector('.entry-form');
+var $title = document.querySelector('.title');
+var $notes = document.querySelector('.notes');
+var $ul = document.querySelector('ul');
+var $noEntries = document.getElementById('noEntries');
+var $newButton = document.querySelector('.newButton');
+var $divHidden = document.querySelector('.hidden');
+var $divContainer = document.querySelectorAll('.container');
+var $divContainerLast = $divContainer[$divContainer.length - 1];
```

Comment:
divContainerLast? What is the purpose of this? The name gives me no hint what it is for

Reply:
renamed to $divContainerEntries